scripts/dev/install-spack.sh

```diff
@@ -29,7 +29,7 @@ info "This script is running from ${SCRIPT_DIR}"
 # Clone Spack and repositories
 ###############################################################################
 
-if [ -n "${SPACK_ROOT}" ]; then
+if [ ! -n "${SPACK_ROOT}" ]; then
```

Comment:
Good catch -- I fixed this in the original one (which I clumsily copied, hence the older `File` name above):
```suggestion
if [ -z "${SPACK_ROOT}" ]; then
```